patterns/2-structured/project-setup/communication-tooling.md

```diff
@@ -20,7 +20,7 @@ to that component. Even when it happens in writing, communication happens in a
 ## Problem
 
 A team is open to receiving contributions from downstream users of their
-component. Coordination and communication happens in an ad-hoc fashion though
+component. Coordination and communication happens in an ad-hoc fashion through
```

Comment:
```suggestion
component. Coordination and communication happens in an ad-hoc fashion though
```

When reading the whole sentence, I believe the authors did mean "though" in this instance.
Maybe there is a comma missing right after "though"?